usr/local/www/reboot.php

```diff
@@ -49,24 +52,33 @@
 $pgtitle = array(gettext("Diagnostics"),gettext("Reboot System"));
 include("head.inc");
 
+if (stristr($_POST['Submit'], gettext("Yes"))) {
+	?><meta http-equiv=\"refresh\" content=\"70;url=/\"> <?php
```

Comment:
There is unneeded escaping now, can you change this to:

```php 
if (...) {
?>
	<meta http-equiv="refresh" content="70;url=/">
	<div class="alert alert-success" role="alert">
		<?=gettext("The system is rebooting now. This may take one minute or so.")?>
	</div>
<?
```